website/docs/mocking.md

```diff
@@ -483,7 +483,7 @@ Set a field value via graph traversal (nested set):
 ```ts
 store.get('Query', 'ROOT', {
   viewer: {
-    name: 'Alexamdre',
+    name: 'Alexandre',
```

Comment:
This is odd, because it seems to reference the `store.set(...)` line above, but, it doesn't.